akka-cluster-tools/src/main/scala/akka/cluster/pubsub/protobuf/DistributedPubSubMessageSerializer.scala

```diff
@@ -98,7 +98,8 @@ private[akka] class DistributedPubSubMessageSerializer(val system: ExtendedActor
         readChunk()
     }
 
-    readChunk()
+    try readChunk()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/vasseurjb/akka/file/2046293194/issues/source?bid=3132306&fileBranchId=3132365#l101)